models/cycle_gan_semantic_model.py

```diff
@@ -53,85 +49,32 @@ def modify_commandline_options(parser, is_train=True):
         return parser
     
     def __init__(self, opt):
-        BaseModel.__init__(self, opt)
+        super().__init__(opt)
```

Comment:
Generally it's good practice to indicate which `super` class you're calling, ie
`super(CycleGanModel, self).__init__(opt)`

Reply:
it won't work, `super().__init__()` is fine :)